src/test/java/com/rocketden/main/service/GameManagementServiceTests.java

```diff
@@ -37,13 +45,40 @@
 
 	@Spy
 	@InjectMocks
-    private GameService gameService;
+    private GameManagementService gameService;
     
     // Predefine user and room attributes.
     private static final String NICKNAME = "rocket";
     private static final String NICKNAME_2 = "rocketrocket";
     private static final String ROOM_ID = "012345";
 
+    @Test
+	public void addGetAndRemoveGame() {
+    	// Initially, room doesn't exist
```

Comment:
nit: Move comments forward by tab.

Reply:
Just reindented the file although idk if that actually fixed it or not - lmk if not

Reply:
Yes, this fixed it -- I'll resolve.